content/20.treatments.md

```diff
@@ -11,48 +11,97 @@ In the following sections, we critically appraise the literature surrounding the
--Other diseases with similar symptoms include XYZ but these diseases may be different because...
--Given what we know about the mechanisms of the virus and why it produces the symptoms we see, are there drugs or categories of intervention that might be relevant?
-
-So far, some strategies for reducing the severity of symptoms have included...
-
-#### Symptom Management Approach 1
-
-Brief background on the therapeutic.
+The clinical picture of SARS-CoV-2 infection differs dramatically between individuals.
+Some are asymptomatic.
+Others experience mild COVID-19 symptoms, such as cough, sore throat and fever, while the most severe cases of COVID-19 include severe pneumonia with respiratory failure which can lead to death. 
+Vaccines are one avenue to mitigate harm from viral pathogens, but in the case of a rapidly growing pandemic the longer timeframe of vaccine development and distribution means that there can be a key for treatments that palliate symptoms to avoid the most severe outcomes from infection. 
+Afterwards, the vaccine will face another challenge. 
+Currently, treatments are going in two different directions.
+On one side, efforts are focusing on the development of new drugs which can cure the symptoms.
+In this section we will describe and analyse the most recent developed drugs that are undergoing new trials, as well as  to face the Corona virus disease. 
+
+#### Tocilizumab
+
+A recent study carried out on a sample of 191 Covid-19 patients from Wuhan shows that the blood of patients who faced death contained about 74% more interleukin-6 (IL-6) compared to the survivors [@doi:ggnxb3].
+This suggests that these individuals may be experiencing a "cytokine storm", which refers to an inflammatory response out of control.
+In this context IL-6 plays a key role [@doi:10.1128/mmbr.05015-11].
+IL-6 is a pro-inflammatory cytokine belonging to the family of interleukins, immune system regulators, primarily responsible for immune cell differentiation. Specifically, they promote the differentiation of activated B cells into immunoglobulin-producing plasma cells, as well as they represent the growth factor for hybridoma and myeloma cells. In addition, IL-6 also induces differentiation of naïve CD4+ T cells into effector T-cell subsets [10.1158/2326-6066].
+In this way interleukins regulate both pro- and anti-inflammatory response.
+In this context, the observation of elevated IL-6 in patients who died may reflect an over-production of proinflammatory interleukins. 
+In a healthy situation the lung respiratory epithelium together with alveolar macrophages limits the activation of the immune system as well as the alveolar spaces, ensuring homeostasis.
+It has been already shown that high level of IL-6, during the infection of other Corona viruses (influenza, SARS COV), lead to the high concentration of other cytokines, such as Tumor Necrosis Factor (TNF) and IL-1𝛽.
```

Comment:
This is one of the papers: 
Up-regulation of IL-6 and TNF-α induced by SARS-coronavirus spike protein in murine macrophages via NF-κB pathway [doi.org/10.1016/j.virusres.2007.02.007]
Here another reference:
Expression of elevated levels of pro‐inflammatory cytokines in SARS‐CoV‐infected ACE2+ cells in SARS patients: relation to the acute lung injury and pathogenesis of SARS [doi.org/10.1002/path.2067]. I thought I already added them yesterday, couldn't you see them? Do these ref look fine to you? I think these results are important in the context tocilizumab use, because IL-1Beta is crucial to the citokine storm reaction: "IL-1beta is a key cytokine driving proinflammatory activity in bronchoalveolar lavage fluid of patients with lung injury", from the paper: into the eye of the citokine storm. What do you think?

Reply:
I wasn't seeing them before, but I'm updating it now!